bucket/falconx.json

```
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://chrisandriessen.nl/web/FalconX.html",
+    "description": "Center Windows taskbar icons with a variety of animations and options.",
+    "version": "1.3.3.0",
+    "license": "MIT",
+    "url": "https://github.com/ChrisAnd1998/FalconX-Center-Taskbar/releases/download/1.3.3.0/FalconX_01.12.2019_1.3.3.0.zip",
+    "hash": "1f435f81d6dfb5e27494af4b4ecbc7eb39d28672065429eb0346d9d8eb9269bf",
+    "bin": "FalconX.exe",
+    "shortcuts": [
+        [
+            "FalconX.exe",
+            "FalconX"
+        ]
+    ],
+    "checkver": {
+        "github:": "https://github.com/ChrisAnd1998/FalconX-Center-Taskbar",
```

Sorry... I missed colon

```suggestion
        "github": "https://github.com/ChrisAnd1998/FalconX-Center-Taskbar",
```

---

Oh... my fault, really sorry... 😅